### ISSUE TITLE
Using up memory without doing a search

### ISSUE THREAD

**Issue by mmc46** [jefferislab/NBLAST_on-the-fly#88]
Just to make a note of what I noticed this week.

While navigating between the tabs, without doing a search, is using up over 800Mb of browser memory. 
I noticed this on Firefox (46.0.1), Greg saw the same with Chrome.

I'm attaching the log of process sampling.

[Sample of Firefox_nblast_online_13052016.txt](https://github.com/jefferislab/NBLAST_on-the-fly/files/263509/Sample.of.Firefox_nblast_online_13052016.txt)

**Comment by ajdm** [comment]
I've been unable to reproduce this just now. Do you still experience this?

**Comment by mmc46** [comment]
Yes, as soon as I open the page Firefox 47.0 goes from ~20% CPU to 60-70%.
